app/controllers/questions/comments_controller.rb

```diff
@@ -0,0 +1,13 @@
+class Questions::CommentsController < CommentsController
```

Comment:
@jerflan - We have a comment controller for Questions as well. The generalized comments controller handles most everything though